docs/self-hosting/fips-deployment.md

```
@@ -0,0 +1,172 @@
+# 1. Confirm the host is in FIPS mode (above).
+# 2. Prepare the environment file.
+cp variables.env .env
+#    Edit at least: DOMAIN_NAME, WEB_URL, SECRET_KEY, MACHINE_SIGNATURE.
+
+# 3. Bring the stack up.
+docker compose -f docker-compose-fips.yml up -d
+```
+
+Each container logs its posture on startup:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)